src/utils.py

```diff
@@ -1667,15 +1668,6 @@ def string_to_python_object(value: str) -> Any:
     return value
 
 
-def print_args(args: argparse.Namespace) -> None:
```

Comment:
are we wiping these printouts? (fine with me)

Reply:
yeah